sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Helpers/VisualBasicSyntaxHelper.cs

```diff
@@ -103,10 +103,12 @@ public static StatementSyntax GetSucceedingStatement(this StatementSyntax curren
 
         public static SyntaxToken? GetMethodCallIdentifier(this InvocationExpressionSyntax invocation)
         {
-           if (invocation == null)
+            if (invocation == null ||
+                invocation.Expression == null)
```

Comment:
This fixes the exceptions from ThreadResumeOrSuspendShouldNotBeCalled and DangerousGetHandleShouldNotBeCalled from our cirrus ci job.